src/app/utils/chromeStorage.ts

```diff
@@ -19,35 +19,35 @@ class ChromeStorage {
     return runtimeMap.get(this).lastError;
   }
 
-  setItem(key: string, item: any): Promise<any> {
+  setItem(key: string, item: any): Promise<void> {
     return new Promise((resolve, reject) => {
       this.getDriver().set({ [key]: item }, () => {
         if (this.hasError()) {
           return reject(this.getError());
         }
-        return resolve(true);
+        return resolve();
       });
     });
   }
 
-  getItem<T = any>(key: string): Promise<T> {
+  getItem<T = any, D = undefined>(key: string, defaultValue?: D): Promise<T | D> {
```

Comment:
By default, the storage interface returns an empty object if the key is not present, resulting in a response of undefined. The Seed Vault expects null though, so I had to add this default value.